main.go

```diff
@@ -99,7 +107,10 @@ func main() {
 	// klog.Background will automatically use the right logger.
 	ctrl.SetLogger(klog.Background())
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	restConfig := ctrl.GetConfigOrDie()
+	restConfig.QPS = float32(restConfigQPS)
```

Comment:
RestConfigQPS is defined as float64 on L49, why do we convert it to float32?

Reply:
Good question. For some reason the QPS field in the rest config is float32 so that is what we need. However, the `flag` library doesn't have a `Float32Var`, only `Float64Var`, so I went with a float64 variable that I convert.

In CAPI and CAPM3 we use another flag library on top of `flag` that *has* a `Float32Var` though. So there we [didn't have to do the conversion](https://github.com/Nordix/cluster-api/blob/ba34136c11580c8cba752a3dd1a408cd8b2a7769/bootstrap/kubeadm/main.go#L128).